### ISSUE TITLE
[rutracker] Unsupported POST Content-Type: application/x-www-form-urlencoded (Config)

### ISSUE THREAD

**Issue by Pheggas** [Jackett/Jackett#13034]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
Loading Native indexers ...
Using HTTP Client: HttpWebClient2
Using FlareSolverr: http://172.17.0.4:8191
Using proxy: Disabled
App config/log directory: /config/Jackett
ThreadPool MaxThreads: 32767 workerThreads, 1000 completionPortThreads
Running in Docker: Yes (image build: v0.20.681-ls58)
File /etc/issue: Ubuntu 18.04.6 LTS \n \l
Jackett variant: CoreLinuxAmdx64
OS version: Unix 5.13.0.27 (64bit OS) (64bit process)
Environment version: 6.0.3 (/app/Jackett/)
Starting Jackett v0.20.681
```

### Description

I'm getting a bit annoyed and angry as i had Jackett on Linux on bare metal. From there i found out jackett doesn't have any proper backup solution so i'm forced to do it manually (copy .config/Jackett file etc to copy indexers and settings..). I did it and then uninstalled Jackett via [this issue](https://github.com/Jackett/Jackett/issues/6169). I set it back up as docker container to avoid any obsticles between FlareSolverr (as i need FlareSolverr to access the RuTracker). In Jackett i pasted the IP, it saved the settings so i'm assuming  FlareSolverr and Jackett can communicate (as they're both in bridge network in docker). I headed into add indexer (as those indexers who aren't open will not appear in the indexer list), added RuTracker, entered credentials and found: (look at Logged error message section)

PS: replacing https with http doesn't work as mentioned in #12696

### Logged Error Messages

```text
Error message that appears after clicking Okay button: Unsupported POST Content-Type: application/x-www-form-urlencoded

And the log:
FlareSolverrSharp.Exceptions.FlareSolverrException: Unsupported POST Content-Type: application/x-www-form-urlencoded
   at FlareSolverrSharp.Solvers.FlareSolverr.GenerateFlareSolverrRequest(HttpRequestMessage request, String sessionId)
   at FlareSolverrSharp.Solvers.FlareSolverr.Solve(HttpRequestMessage request, String sessionId)
   at FlareSolverrSharp.ClearanceHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)
   at Jackett.Common.Utils.Clients.HttpWebClient2.Run(WebRequest webRequest) in /home/vsts/work/1/s/src/Jackett.Common/Utils/Clients/HttpWebClient2.cs:line 178
   at Jackett.Common.Utils.Clients.WebClient.GetResultAsync(WebRequest request) in /home/vsts/work/1/s/src/Jackett.Common/Utils/Clients/WebClient.cs:line 186
   at Jackett.Common.Indexers.BaseWebIndexer.RequestLoginAndFollowRedirect(String url, IEnumerable`1 data, String cookies, Boolean returnCookiesFromFirstCall, String redirectUrlOverride, String referer, Boolean accumulateCookies, Dictionary`2 headers) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 621
   at Jackett.Common.Indexers.RuTracker.ApplyConfiguration(JToken configJson) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/RuTracker.cs:line 1428
   at Jackett.Server.Controllers.IndexerApiController.UpdateConfig(ConfigItem[] config) in /home/vsts/work/1/s/src/Jackett.Server/Controllers/IndexerApiController.cs:line 97
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/vsts/work/1/s/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 61
```

### Screenshots

![image](https://user-images.githubusercontent.com/9994511/157513413-1ce7c462-60de-4dad-bee1-71b1c9c8ca3b.png)
![image](https://user-images.githubusercontent.com/9994511/157512056-f8e942d8-8b92-40ee-97b0-82165dee6e7d.png)

**Comment by ilike2burnthing** [comment]
Working ok here, not sure why you're getting that error.

Post an enhanced log of the issue - https://github.com/Jackett/Jackett/wiki/Troubleshooting#how-to-provide-a-jackett-enhanced-log-for-a-ticket

The wider issue is FlareSolverr's - https://github.com/FlareSolverr/FlareSolverr/issues/77

**Comment by Pheggas** [comment]
No problem. Here you go:

2022-03-10 07:30:55 | Debug | Executed controller factory for controller Jackett.Server.Controllers.ServerConfigurationController (jackett)
-- | -- | --
2022-03-10 07:30:55 | Debug | Executing controller factory for controller Jackett.Server.Controllers.ServerConfigurationController (jackett)
2022-03-10 07:30:55 | Debug | Authorization was successful.
2022-03-10 07:30:55 | Debug | Execution plan of result filters (in the following order): None
2022-03-10 07:30:55 | Debug | Execution plan of exception filters (in the following order): None
2022-03-10 07:30:55 | Debug | Execution plan of action filters (in the following order): Microsoft.AspNetCore.Mvc.Filters.ControllerActionFilter (Order: -2147483648), Microsoft.AspNetCore.Mvc.ModelBinding.UnsupportedContentTypeFilter (Order: -3000), Microsoft.AspNetCore.Mvc.Filters.ResponseCacheFilter
2022-03-10 07:30:55 | Debug | Execution plan of resource filters (in the following order): None
2022-03-10 07:30:55 | Debug | Execution plan of authorization filters (in the following order): Microsoft.AspNetCore.Mvc.Authorization.AuthorizeFilter
2022-03-10 07:30:55 | Info | Route matched with {action = "Logs", controller = "ServerConfiguration"}. Executing controller action with signature System.Collections.Generic.List`1[Jackett.Common.Models.CachedLog] Logs() on controller Jackett.Server.Controllers.ServerConfigurationController (jackett).
2022-03-10 07:30:55 | Info | Executing endpoint 'Jackett.Server.Controllers.ServerConfigurationController.Logs (jackett)'
2022-03-10 07:30:55 | Debug | Request matched endpoint 'Jackett.Server.Controllers.ServerConfigurationController.Logs (jackett)'
2022-03-10 07:30:55 | Debug | Endpoint 'Jackett.Server.Controllers.ServerConfigurationController.Logs (jackett)' with route pattern 'api/v2.0/server/Logs' is valid for the request path '/api/v2.0/server/logs'
2022-03-10 07:30:55 | Debug | 1 candidate(s) found for the request path '/api/v2.0/server/logs'
2022-03-10 07:30:55 | Debug | AuthenticationScheme: Cookies was successfully authenticated.
2022-03-10 07:30:55 | Debug | Connection id "0HMG1VJ79HQG8" stopped.
2022-03-10 07:30:55 | Debug | The request path /api/v2.0/server/logs does not match a supported file type
2022-03-10 07:30:55 | Debug | Request is continuing in applying rules. Current url is http://192.168.1.12:9117/api/v2.0/server/logs?_=1646893793615
2022-03-10 07:30:55 | Debug | Connection id "0HMG1VJ79HQG8" sending FIN because: "The client closed the connection."
2022-03-10 07:30:55 | Debug | Request is continuing in applying rules. Current url is http://192.168.1.12:9117/api/v2.0/server/logs?_=1646893793615
2022-03-10 07:30:55 | Debug | Connection id "0HMG1VJ79HQG8" disconnecting.
2022-03-10 07:30:55 | Debug | Request is continuing in applying rules. Current url is http://192.168.1.12:9117/api/v2.0/server/logs?_=1646893793615
2022-03-10 07:30:55 | Info | Request starting HTTP/1.1 GET http://192.168.1.12:9117/api/v2.0/server/logs?_=1646893793615 - -
2022-03-10 07:30:55 | Debug | Connection id "0HMG1VJ79HQG8" received FIN.
2022-03-10 07:30:51 | Info | Request finished HTTP/1.1 POST http://192.168.1.12:9117/api/v2.0/indexers/rutracker/config application/json 414 - 500 - application/json 73.1908ms
2022-03-10 07:30:51 | Debug | Connection id "0HMG1VJ79HQG6" completed keep alive response.
2022-03-10 07:30:51 | Debug | The response will be compressed with 'gzip'.
2022-03-10 07:30:51 | Error | FlareSolverrSharp.Exceptions.FlareSolverrException: Unsupported POST Content-Type: application/x-www-form-urlencoded    at FlareSolverrSharp.Solvers.FlareSolverr.GenerateFlareSolverrRequest(HttpRequestMessage request, String sessionId)    at FlareSolverrSharp.Solvers.FlareSolverr.Solve(HttpRequestMessage request, String sessionId)    at FlareSolverrSharp.ClearanceHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)    at System.Net.Http.HttpClient.<SendAsync>g__Core\|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)    at Jackett.Common.Utils.Clients.HttpWebClient2.Run(WebRequest webRequest) in /home/vsts/work/1/s/src/Jackett.Common/Utils/Clients/HttpWebClient2.cs:line 178    at Jackett.Common.Utils.Clients.WebClient.GetResultAsync(WebRequest request) in /home/vsts/work/1/s/src/Jackett.Common/Utils/Clients/WebClient.cs:line 209    at Jackett.Common.Indexers.BaseWebIndexer.RequestLoginAndFollowRedirect(String url, IEnumerable`1 data, String cookies, Boolean returnCookiesFromFirstCall, String redirectUrlOverride, String referer, Boolean accumulateCookies, Dictionary`2 headers) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 621    at Jackett.Common.Indexers.RuTracker.ApplyConfiguration(JToken configJson) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/RuTracker.cs:line 1428    at Jackett.Server.Controllers.IndexerApiController.UpdateConfig(ConfigItem[] config) in /home/vsts/work/1/s/src/Jackett.Server/Controllers/IndexerApiController.cs:line 97    at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited\|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited\|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)    at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited\|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)    at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited\|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)    at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged\|17_1(ResourceInvoker invoker)    at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged\|17_1(ResourceInvoker invoker)    at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask\|6_0(Endpoint endpoint, Task requestTask, ILogger logger)    at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)    at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/vsts/work/1/s/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 61
2022-03-10 07:30:51 | Info | Executed endpoint 'Jackett.Server.Controllers.IndexerApiController.UpdateConfig (jackett)'
2022-03-10 07:30:51 | Info | Executed action Jackett.Server.Controllers.IndexerApiController.UpdateConfig (jackett) in 59.6351ms
2022-03-10 07:30:51 | Debug | WebClient(HttpWebClient2).GetResultAsync(Method: POST Url: https://rutracker.org/forum/login.php PostData: {login_username=Pheggas, login_password=depesak100, login=Login} RawBody: )
2022-03-10 07:30:51 | Debug | CACHE IsCacheEnabled => false
2022-03-10 07:30:51 | Debug | Done attempting to validate the bound parameter 'config' of type 'Jackett.Common.Models.DTO.ConfigItem[]'.
2022-03-10 07:30:51 | Debug | Attempting to validate the bound parameter 'config' of type 'Jackett.Common.Models.DTO.ConfigItem[]' ...
2022-03-10 07:30:51 | Debug | Done attempting to bind parameter 'config' of type 'Jackett.Common.Models.DTO.ConfigItem[]'.
2022-03-10 07:30:51 | Debug | Done attempting to bind parameter 'config' of type 'Jackett.Common.Models.DTO.ConfigItem[]'.
2022-03-10 07:30:51 | Debug | Connection id "0HMG1VJ79HQG6", Request id "0HMG1VJ79HQG6:00000020": done reading request body.
2022-03-10 07:30:51 | Debug | Connection id "0HMG1VJ79HQG6", Request id "0HMG1VJ79HQG6:00000020": started reading request body.
2022-03-10 07:30:51 | Debug | Selected input formatter 'Microsoft.AspNetCore.Mvc.Formatters.NewtonsoftJsonInputFormatter' for content type 'application/json'.
2022-03-10 07:30:51 | Debug | Rejected input formatter 'Microsoft.AspNetCore.Mvc.Formatters.NewtonsoftJsonPatchInputFormatter' for content type 'application/json'.
2022-03-10 07:30:51 | Debug | Attempting to bind parameter 'config' of type 'Jackett.Common.Models.DTO.ConfigItem[]' using the name '' in request data ...
2022-03-10 07:30:51 | Debug | Attempting to bind parameter 'config' of type 'Jackett.Common.Models.DTO.ConfigItem[]' ...
2022-03-10 07:30:51 | Debug | Executed controller factory for controller Jackett.Server.Controllers.IndexerApiController (jackett)
2022-03-10 07:30:51 | Debug | Executing controller factory for controller Jackett.Server.Controllers.IndexerApiController (jackett)
2022-03-10 07:30:51 | Debug | Authorization was successful.
2022-03-10 07:30:51 | Debug | Execution plan of result filters (in the following order): None
2022-03-10 07:30:51 | Debug | Execution plan of exception filters (in the following order): None
2022-03-10 07:30:51 | Debug | Execution plan of action filters (in the following order): Microsoft.AspNetCore.Mvc.Filters.ControllerActionFilter (Order: -2147483648), Microsoft.AspNetCore.Mvc.ModelBinding.UnsupportedContentTypeFilter (Order: -3000), Microsoft.AspNetCore.Mvc.Filters.ResponseCacheFilter, Jackett.Server.Controllers.RequiresIndexer
2022-03-10 07:30:51 | Debug | Execution plan of resource filters (in the following order): None
2022-03-10 07:30:51 | Debug | Execution plan of authorization filters (in the following order): Microsoft.AspNetCore.Mvc.Authorization.AuthorizeFilter
2022-03-10 07:30:51 | Info | Route matched with {action = "UpdateConfig", controller = "IndexerApi"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] UpdateConfig(Jackett.Common.Models.DTO.ConfigItem[]) on controller Jackett.Server.Controllers.IndexerApiController (jackett).
2022-03-10 07:30:51 | Info | Executing endpoint 'Jackett.Server.Controllers.IndexerApiController.UpdateConfig (jackett)'
2022-03-10 07:30:51 | Debug | Request matched endpoint 'Jackett.Server.Controllers.IndexerApiController.UpdateConfig (jackett)'
2022-03-10 07:30:51 | Debug | Endpoint 'Jackett.Server.Controllers.IndexerApiController.UpdateConfig (jackett)' with route pattern 'api/v2.0/indexers/{indexerId?}/Config' is valid for the request path '/api/v2.0/indexers/rutracker/config'
2022-03-10 07:30:51 | Debug | 1 candidate(s) found for the request path '/api/v2.0/indexers/rutracker/config'
2022-03-10 07:30:51 | Debug | AuthenticationScheme: Cookies was successfully authenticated.
2022-03-10 07:30:51 | Debug | POST requests are not supported
2022-03-10 07:30:51 | Debug | Request is continuing in applying rules. Current url is http://192.168.1.12:9117/api/v2.0/indexers/rutracker/config
2022-03-10 07:30:51 | Debug | Request is continuing in applying rules. Current url is http://192.168.1.12:9117/api/v2.0/indexers/rutracker/config
2022-03-10 07:30:51 | Debug | Request is continuing in applying rules. Current url is http://192.168.1.12:9117/api/v2.0/indexers/rutracker/config
2022-03-10 07:30:51 | Info | Request starting HTTP/1.1 POST http://192.168.1.12:9117/api/v2.0/indexers/rutracker/config application/json 414
2022-03-10 07:30:48 | Info | Request finished HTTP/1.1 GET http://192.168.1.12:9117/api/v2.0/indexers/rutracker/config?_=1646893793614 - - - 200 - application/json;+charset=utf-8 3177.4171ms
2022-03-10 07:30:48 | Debug | Connection id "0HMG1VJ79HQG6" completed keep alive response.
2022-03-10 07:30:48 | Info | Executed endpoint 'Jackett.Server.Controllers.IndexerApiController.Config (jackett)'
2022-03-10 07:30:48 | Info | Executed action Jackett.Server.Controllers.IndexerApiController.Config (jackett) in 3173.9772ms
2022-03-10 07:30:48 | Debug | The response will be compressed with 'gzip'.
2022-03-10 07:30:48 | Info | Executing OkObjectResult, writing value of type 'Newtonsoft.Json.Linq.JArray'.
2022-03-10 07:30:48 | Debug | Selected output formatter 'Microsoft.AspNetCore.Mvc.Formatters.NewtonsoftJsonOutputFormatter' and content type 'application/json' to write the response.
2022-03-10 07:30:48 | Debug | Attempting to select the first formatter in the output formatters list which can write the result.
2022-03-10 07:30:48 | Debug | Attempting to select an output formatter without using a content type as no explicit content types were specified for the response.
2022-03-10 07:30:48 | Debug | No information found on request to perform content negotiation.
2022-03-10 07:30:48 | Debug | List of registered output formatters, in the following order: Microsoft.AspNetCore.Mvc.Formatters.HttpNoContentOutputFormatter, Microsoft.AspNetCore.Mvc.Formatters.StringOutputFormatter, Microsoft.AspNetCore.Mvc.Formatters.StreamOutputFormatter, Microsoft.AspNetCore.Mvc.Formatters.NewtonsoftJsonOutputFormatter
2022-03-10 07:30:48 | Error | Error loading configuration: FlareSolverrSharp.Exceptions.FlareSolverrException: Error connecting to FlareSolverr server: System.Net.Http.HttpRequestException: No route to host (172.17.0.4:8191)  ---> System.Net.Sockets.SocketException (113): No route to host    at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)    at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)    at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation\|277_0(AwaitableSocketAsyncEventArgs saea, ValueTask connectTask, CancellationToken cancellationToken)    at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)    --- End of inner exception stack trace ---    at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)    at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)    at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)    at System.Net.Http.HttpConnectionPool.AddHttp11ConnectionAsync(HttpRequestMessage request)    at System.Threading.Tasks.TaskCompletionSourceWithCancellation`1.WaitWithCancellationAsync(CancellationToken cancellationToken)    at System.Net.Http.HttpConnectionPool.GetHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)    at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)    at System.Net.Http.DiagnosticsHandler.SendAsyncCore(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)    at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)    at System.Net.Http.HttpClient.<SendAsync>g__Core\|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)    at FlareSolverrSharp.Solvers.FlareSolverr.<>c__DisplayClass10_0.<<SendFlareSolverrRequest>b__0>d.MoveNext()    at FlareSolverrSharp.Solvers.FlareSolverr.<>c__DisplayClass10_0.<<SendFlareSolverrRequest>b__0>d.MoveNext() --- End of stack trace from previous location ---    at FlareSolverrSharp.Utilities.SemaphoreLocker.LockAsync[T](Func`1 worker)    at FlareSolverrSharp.Solvers.FlareSolverr.SendFlareSolverrRequest(HttpContent flareSolverrRequest)    at FlareSolverrSharp.Solvers.FlareSolverr.Solve(HttpRequestMessage request, String sessionId)    at FlareSolverrSharp.ClearanceHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)    at System.Net.Http.HttpClient.<SendAsync>g__Core\|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)    at Jackett.Common.Utils.Clients.HttpWebClient2.Run(WebRequest webRequest) in /home/vsts/work/1/s/src/Jackett.Common/Utils/Clients/HttpWebClient2.cs:line 178    at Jackett.Common.Utils.Clients.WebClient.GetResultAsync(WebRequest request) in /home/vsts/work/1/s/src/Jackett.Common/Utils/Clients/WebClient.cs:line 209    at Jackett.Common.Indexers.BaseWebIndexer.RequestWithCookiesAsync(String url, String cookieOverride, RequestType method, String referer, IEnumerable`1 data, Dictionary`2 headers, String rawbody, Nullable`1 emulateBrowser) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 607    at Jackett.Common.Indexers.RuTracker.GetConfigurationForSetup() in /home/vsts/work/1/s/src/Jackett.Common/Indexers/RuTracker.cs:line 1382
2022-03-10 07:30:45 | Debug | WebClient(HttpWebClient2).GetResultAsync(Method: GET Url: https://rutracker.org/forum/login.php)
2022-03-10 07:30:45 | Debug | Executed controller factory for controller Jackett.Server.Controllers.IndexerApiController (jackett)
2022-03-10 07:30:45 | Debug | Executing controller factory for controller Jackett.Server.Controllers.IndexerApiController (jackett)
2022-03-10 07:30:45 | Debug | Authorization was successful.
2022-03-10 07:30:45 | Debug | Execution plan of result filters (in the following order): None
2022-03-10 07:30:45 | Debug | Execution plan of exception filters (in the following order): None
2022-03-10 07:30:45 | Debug | Execution plan of action filters (in the following order): Microsoft.AspNetCore.Mvc.Filters.ControllerActionFilter (Order: -2147483648), Microsoft.AspNetCore.Mvc.ModelBinding.UnsupportedContentTypeFilter (Order: -3000), Microsoft.AspNetCore.Mvc.Filters.ResponseCacheFilter, Jackett.Server.Controllers.RequiresIndexer
2022-03-10 07:30:45 | Debug | Execution plan of resource filters (in the following order): None
2022-03-10 07:30:45 | Debug | Execution plan of authorization filters (in the following order): Microsoft.AspNetCore.Mvc.Authorization.AuthorizeFilter
2022-03-10 07:30:45 | Info | Route matched with {action = "Config", controller = "IndexerApi"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] Config() on controller Jackett.Server.Controllers.IndexerApiController (jackett).
2022-03-10 07:30:45 | Info | Executing endpoint 'Jackett.Server.Controllers.IndexerApiController.Config (jackett)'
2022-03-10 07:30:45 | Debug | Request matched endpoint 'Jackett.Server.Controllers.IndexerApiController.Config (jackett)'
2022-03-10 07:30:45 | Debug | Endpoint 'Jackett.Server.Controllers.IndexerApiController.Config (jackett)' with route pattern 'api/v2.0/indexers/{indexerId?}/Config' is valid for the request path '/api/v2.0/indexers/rutracker/config'
2022-03-10 07:30:45 | Debug | 1 candidate(s) found for the request path '/api/v2.0/indexers/rutracker/config'
2022-03-10 07:30:45 | Debug | AuthenticationScheme: Cookies was successfully authenticated.
2022-03-10 07:30:45 | Debug | The request path /api/v2.0/indexers/rutracker/config does not match a supported file type
2022-03-10 07:30:45 | Debug | Request is continuing in applying rules. Current url is http://192.168.1.12:9117/api/v2.0/indexers/rutracker/config?_=1646893793614
2022-03-10 07:30:45 | Debug | Request is continuing in applying rules. Current url is http://192.168.1.12:9117/api/v2.0/indexers/rutracker/config?_=1646893793614
2022-03-10 07:30:45 | Debug | Request is continuing in applying rules. Current url is http://192.168.1.12:9117/api/v2.0/indexers/rutracker/config?_=1646893793614
2022-03-10 07:30:45 | Info | Request starting HTTP/1.1 GET http://192.168.1.12:9117/api/v2.0/indexers/rutracker/config?_=1646893793614 - -

**Comment by garfield69** [comment]
do you get the same issue of you switch to using `https://rutracker.net/` ?
While the .org invokes flaresolverr, the .net does not for me, wonder if its the same for you?

**Comment by garfield69** [comment]
@ilike2burnthing  I can replicate this issue.
- start with the indexer using .org
- switch to using .net, save and test
- switch back to using .org
- get the post error
- note that flaresolverr does not log any activity (even with log_level=debug and log_html=true)
- close the config
- add the indexer back with .org and save
- working again for me.

**Comment by Pheggas** [comment]
> do you get the same issue if you switch to using `https://rutracker.net/` ? While the .org invokes flaresolverr, the .net does not for me, wonder if its the same for you?

Works well with this domain!
Issure resolved, can we close this issue?

**Comment by garfield69** [comment]
I'd like to keep it open until @ilike2burnthing can determine if there is anything useful we can provide to @ngosang to try to prevent the mysterious post error. We will take care of the ticket closure ourselves. Thanks.

**Comment by Pheggas** [comment]
I tried using using it with Lidarr (as i have music library) and found out it can't provide any results. When i try searching directly in Jackett, it works fine tho..

**Comment by Pheggas** [comment]
As i have Lidarr as music manager, i tried to search RuTracker there too. But found nothing. Headed into Jackett and searched for exactly the same album (and exactly the same string that Lidarr was searching for) and found all the results. However, in jackett, it shows that those torrents has Category "Audio". So i checked only audio categories (to see if they're really in that category) and clicked search. **Nothing found**. Do you have any reason for this?

**Comment by ilike2burnthing** [comment]
Yep, I can replicate it using https://github.com/Jackett/Jackett/issues/13034#issuecomment-1063731827, but just reconfiguring using *.org didn't cause that problem. Very weird.

As for not finding results when you specify categories, RuTracker limits searches to a maximum of 200 categories, and if you select Jackett's `Audio` category, it includes more than that (~300). In your case, the categories for your search results were outside of the first 200 categories.

In short, you'll need to select the relevant `100000+` categories, rather than using the `30*0` ones. This list may make it easier for you, e.g. tracker category `409` will be indexer category `100409` - https://github.com/Jackett/Jackett/blob/59208fcf44f82d6d8a10ef70ca7a6caa6793bdaf/src/Jackett.Common/Indexers/RuTracker.cs#L771-L1077

**Comment by heartbleeded** [comment]
How could I switch to .org domain ? Changed "Site Link" in its configuration from `.ru` to `.org` does not work for me.

**Comment by garfield69** [comment]
@heartbleeded  there are two web sites and therefore 2 indexers.
The indexer called RuTracker.ru which has the site link `http://rutracker.ru/` is a public site, and is run by a different group from the indexer called RuTracker, which is a semi-private site that has two domains available `https://rutracker.org/` and `https://rutracker.net/`
The two site have different engines, and the indexer for one cannot use the domain for the other.

**Comment by Pheggas** [comment]
> In short, you'll need to select the relevant 100000+ categories, rather than using the 30*0 ones. 

That would be okay but how would i do it if RuTracker limit it's categories to 200?
Also, i tried to check all the categories in both - Jackett and Lidarr but both got error that the link is too long (which i'm assuming it's the protection over 200 categories).

The interesting part is when i had checked some categories in Lidarr that has something to do with music and audio, it found everything i wanted but after switching from .org domain to .net, nothing to be found.

It's clear they have different torrents in different caregories than on .org domain. Is there workaround?

**Comment by ilike2burnthing** [comment]
Just select fewer that 200 categories from the `100000+` range, and do not select any from the `30*0` range.

As far as I've seen *.org and *.net are exactly the same. If you can provide 2 URLs, with the only difference being .org/.net, that have differing results, then it will be worth taking a second look.

**Comment by Pheggas** [comment]
> Just select fewer that 200 categories

It's not possible to cover all the music categories with less than 200 categories limit. I'm downloading and searching for many genres so it's not a way for me

**Comment by garfield69** [comment]
>It's not possible to cover all the music categories with less than 200 categories limit. I'm downloading and searching for many genres so it's not a way for me

there is nothing we can do about that. The limiting factor is that the web site has set a URL length limit. When Jackett builds the search URL and concatenates all the categories, the whole URL must fit within the specified length or be rejected. About 200 categories is the practical limit that can be sent to the site.

**Comment by Pheggas** [comment]
>About 200 categories is the practical limit that can be sent to the site.

And what about splitting it into 2 or 3 requests? So it can easily fit 600 categories in 3 requests.

**Comment by garfield69** [comment]
Not recommended.
Making multiple requests for a single search in these days of heightened DDoS awareness may lead to your IP being blocked or your account suspended for spamming.

**Comment by Pheggas** [comment]
Well, this can be solved by using VPN tunnel for each connection. Let's say Proton VPN.

**Comment by ilike2burnthing** [comment]
Given it's a private tracker, your account would just be banned.

If you want to run multiple Jackett instances and split the categories between them, by all means. Just be warned that your account may be at risk by doing so, and it's not the fault of Jackett if you lose it.

**Comment by Pheggas** [comment]
No problem
